index.html

```diff
@@ -1,17 +1,121 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Address Book</title>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/6.0.0/normalize.min.css">
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;800&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="styles.css">
+  <title>Address Book</title>
 
-  </head>
-  <body>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/6.0.0/normalize.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+  <div class="container">
+
+    <h1>address book</h1>
+
+    <div class="delete_selection">
+      <div>
+        <img src="images/delete.svg" alt="two swords crossed">
+      </div>
+
+      <label for="contact_name">Delete contact<br>from database</label>
+      <p class="extra_info">&#40;Exact contact name match &#41;</p>
```

Comment:
There is no need to use HTML entities here for the parentheses, you can use them directly:
```
(Exact contact name match)
```